sigs/sig-scalability/README.md

```diff
@@ -0,0 +1,23 @@
+# Argo SIG Scalability
+
+Argo SIG Scalability is focused on the scalability and performance of the Argo projects.
+
+The initial focus of the group is on Argo CD, working through the [the Argo CD Scalability Benchmarking proposal](https://github.com/argoproj/argo-cd/pull/12662). When the time comes that other's want to join and focus on another project (e.g. Argo Workflows), [a proposal](https://github.com/argoproj/argoproj/tree/master/proposals) should be created to expand the scope of the SIG.
+
+The goals of this group are to create benchmarking tooling to objectively measure the scaliablity of the projects, and improving [the "scaling up" experience](https://argo-cd.readthedocs.io/en/stable/operator-manual/high_availability/#scaling-up) for end users.
+
+## Meetings
+SIG Scalability meetings are held on the second and forth Wednesday of each month. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com).
+
+The members of the group can be contacted on [the CNCF Slack](https://slack.cncf.io/), in the `#argo-sig-scalability` channel and by using [the `type:scalability` label](https://github.com/argoproj/argo-cd/pulls?q=is%3Apr+is%3Aopen+label%3Atype%3Ascalability) on PRs and issues.
+
+## Members
+<!-- Alphanumeric order based on `Name` -->
```

Comment:
I assume this will have open membership, e.g. 
"Membership is open to any interested person in the Argo community"